week4/assignment/week4_assignment/src/pages/MyPage.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import styled from '@emotion/styled';
+import { useNavigate } from 'react-router-dom';
+import FindHobby from '../pages/MyPage/FindHobby';
+import FixInfo from '../pages/MyPage/FixInfo';
+import { Theme } from '../styles/Themes';
+
+const Mypage = () => {
+  const [viewMode, setViewMode] = useState('hobby');
```

Comment:
```suggestion
  const [viewMode, setViewMode] = useState<'hobby' | 'info'>('hobby');
```

viewMode의 경우 'hobby'또는 'info'값을 가지므로, 위와 같이 타입을 지정해주면 조금 더 안전한 코드를 짤 수 있을 것 같습니다.